components/yaml/introduction.rst

```diff
@@ -214,30 +200,27 @@ changed using the third argument as follows::
             foo: bar
             bar: baz
 
-Invalid Types and Object Serialization
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Advanced Usage: Flags
+---------------------
 
```

Comment:
I thought about adding a table here with the flag, which method it applies to  (linking to a section below) and a short description.